eggbox_potential_sampler/model_based_mco/tests/test_mco_factory.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+# from force_bdss.core_plugins import BasicMCOCommunicator
+from enthought_example.example_mco.example_mco_communicator import \
+    ExampleMCOCommunicator
+from eggbox_potential_sampler.eggbox_plugin import EggboxPlugin
+from eggbox_potential_sampler.model_based_mco.mco_model import (
+    ModelBasedOptimizationMCOModel
+)
+from eggbox_potential_sampler.model_based_mco.mco import (
+    ModelBasedOptimizationMCO
+)
+
+
+class TestModelBasedOptimizationMCOFactory(unittest.TestCase):
+    def setUp(self):
+        self.plugin = EggboxPlugin()
+        self.factory = self.plugin.mco_factories[1]
```

Comment:
Quite a brittle test! Relies on constant ordering of the plugin-defined factory list.